tools/bundle-size-tools/bundle-analysis-app/tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
```

Comment:
I don't know if it matters but just double checking if this is intentional: 
bundle-analysis-app package is now in a subfolder within the package bundle-size-tools?

Reply:
It was intentional, I thought it would be good to put the app along with the tools, but I could move it elsewhere!